src/types.ts

```diff
@@ -134,6 +136,21 @@ export interface SwaggerToTSOptions {
   silent?: boolean;
   /** (optional) OpenAPI version. Must be present if parsing raw schema */
   version?: number;
+  /**
+   * (optional) List of HTTP headers that will be sent with the fetch request to a remote schema. This is
+   * in addition to the authorization header. In some cases, servers require headers such as Accept: application/json
+   * or Accept: text/yaml to be sent in order to figure out how to properly fetch the OpenAPI/Swagger document as code.
+   * These headers will only be sent in the case that the schema URL protocol is of type http or https.
+   */
+  httpHeaders?: Headers;
```

Comment:
This is an opinionated change, but in the previous PR we allowed `httpHeaders?:` to be a polymorphic type, from `null` to a JS object to a `Map`. We then had to implement quite a bit of runtime logic to check to make sure everything lined up correctly.

Instead, I opted to go with just a plain JS object for headers because that’s what gets passed to `fetch()` anyway. Servers are lax in what they accept. And while I completely get the author’s original desire to strongly type their request headers, I believe that’s outside the scope of this library, and could only introduce possible bugs/unnecessary maintenance.